src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -24,27 +32,37 @@ specific language governing permissions and limitations under the License.
 }
 
 .tab.selected {
-  background: var(--theme-tab-active-background);
+  background-position: var(--theme-tab-active-background-position);
+  box-shadow: var(--theme-focus-outline-style);
```

Comment:
One more thing: I see you're using `--theme-focus-outline-style` for the deselected box style. I assume it's because the linter told you to use a variable for this, and went ahead and suggested one for you. It'd prolly be better to have a dedicated variable for this, or even using `stylelint-disable-line` for it, since this isn't something we really want themable (we have `--theme-tab-separator-color` for when a separator needs to show).